package.json

```diff
@@ -27,22 +26,6 @@
   },
   "engineStrict": true,
   "devDependencies": {
-    "@commitlint/cli": "^8.3.5",
-    "@commitlint/config-conventional": "^8.3.4",
```

Comment:
These are the commit lint plugins for husky, do the hooks still work?

Reply:
Sorry commit lint and husky are needed as dev dependencies. Once I did git clean it started compaining.